dspace/config/dspace.cfg

```diff
@@ -1585,6 +1592,13 @@ request.item.type = all
 request.item.helpdesk.override = false
 
 
+#---------------------------------------------------------------#
+#----------ITEM DATABASE STATUS SCRIPT CONFIGURATION------------#
+#---------------------------------------------------------------#
+
+# The max amount of time allowed for an item to be present in solr with predb status without needing a reindex (in ms)
+solr-database-resync.time-until-reindex = 600000
```

Comment:
Tiny thing... based on the code, we should add a comment that says: "When unspecified or commented out, the default is 0".